[BUG] Purge-Command fails if amount=1

/moderation purge messages amount:1

fails with


```
14:19:44 [JDAC EventHandler-Thread 4e34fd64-e4c1-42c6-8d63-9e2c9647dbd1] INFO  c.g.k.j.c.d.handling.EventHandler - Executing interaction "/moderation purge messages" for user "SimuPlays"
14:19:44 [JDAC EventHandler-Thread 4e34fd64-e4c1-42c6-8d63-9e2c9647dbd1] ERROR c.g.k.j.c.d.handling.EventHandler - Interaction execution failed!
java.lang.reflect.InvocationTargetException: null
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
        at java.base/java.lang.reflect.Method.invoke(Unknown Source)
        at com.github.kaktushose.jda.commands.definitions.description.reflective.ReflectiveDescriptor.lambda$method$0(ReflectiveDescriptor.java:55)
        at com.github.kaktushose.jda.commands.definitions.features.internal.Invokable.invoke(Invokable.java:41)
        at com.github.kaktushose.jda.commands.dispatching.handling.EventHandler.invoke(EventHandler.java:104)
        at com.github.kaktushose.jda.commands.dispatching.handling.EventHandler.accept(EventHandler.java:86)
        at com.github.kaktushose.jda.commands.dispatching.Runtime.executeHandler(Runtime.java:102)
        at com.github.kaktushose.jda.commands.dispatching.Runtime.lambda$checkForEvents$1(Runtime.java:91)
        at java.base/java.lang.VirtualThread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: Limit may not be negative or zero
        at net.dv8tion.jda.internal.utils.Checks.positive(Checks.java:196)
        at net.dv8tion.jda.api.entities.MessageHistory$MessageRetrieveAction.limit(MessageHistory.java:579)
        at de.nplay.moderationbot.moderation.commands.BulkDeleteCommands.purgeMessages(BulkDeleteCommands.java:61)
        at de.nplay.moderationbot.moderation.commands.BulkDeleteCommands.purgeMessagesByAmount(BulkDeleteCommands.java:36)
        ... 9 common frames omitted
```

--- a/src/main/java/de/nplay/moderationbot/moderation/commands/BulkDeleteCommands.java
+++ b/src/main/java/de/nplay/moderationbot/moderation/commands/BulkDeleteCommands.java
@@ -33,7 +33,7 @@ public class BulkDeleteCommands {
 
     @SlashCommand(value = "moderation purge messages", desc = "Löscht eine bestimmte Anzahl an Nachrichten gleichzeitig", isGuildOnly = true, enabledFor = Permission.MESSAGE_MANAGE)
     public void purgeMessagesByAmount(CommandEvent event, @Param("Anzahl der Nachrichten die gelöscht werden sollen") @Min(1) @Max(100) int amount) {
-        var deletedMessages = purgeMessages(event, event.getMessageChannel(), event.getMessageChannel().getLatestMessageId(), amount);
+        var deletedMessages = purgeMessages(event, event.getMessageChannel(), event.getMessageChannel().getLatestMessageId(), amount) - 1;
         replyEvent(event, deletedMessages);
     }
 
@@ -58,7 +58,7 @@ public class BulkDeleteCommands {
             );
         } else {
             messageIds.addAll(MessageHistory.getHistoryBefore(channel, pivotMessageId)
-                    .limit(amount - 1)
+                    .limit(amount)
                     .complete()
                     .getRetrievedHistory()
                     .stream()

